### ISSUE TITLE
[Product Quality] Homescreen - Checklist (Ongoing)

### ISSUE THREAD

**Issue by AmyYLee** [mozilla-mobile/fenix#2394]
**Feature Level Checklist for Homescreen**

This is an ongoing list for product quality related bugs for this feature/component. Any non-essential UX issues should be added here (bugs that impact quality of product but do not hinder functionality or accessibility). Please see examples of what is considered essential and non-essential UX bugs [here]<---TBD. Essential UX bugs should be filed separately and triaged by product. Items that are fixed should be checked off by UX (Engineering please ping UX owner when an item is fixed)

- - -
**Animations:** Implement animations 
See https://github.com/mozilla-mobile/fenix/issues/167 for animation details

* [ ] Opening a new tab
* [ ] Closing a tab
* [ ] Opening an existing tab from the home screen
* [ ] tab switching (from reference browser: mozilla-mobile/reference-browser#511)
* [ ] Tapping search bar from home screen
* [x] Private Browsing - Switching the home view from normal to private mode (namely the background gradient) #181
* [x] 3 dot menu Animation #343

[Reporter: Amy Lee, Date May 9 2019]

**Comment by sblatz** [comment]
We are holding off on this work for now as home screen and collections are being redesigned.

**Comment by lime124** [comment]
added homescreen tag and removing needs ux. we'll revisit this when we do more work on homescreen.

**Comment by liuche** [comment]
@vesta0 and @apbitner to go over this list and see what we should do/not do right now, out of the Homescreen design work that's been done.

**Comment by liuche** [comment]
Waiting on UR, should have more insights 8/21

**Comment by liuche** [comment]
@vesta0 were there any insights from the UR from 8/21 that affect this bug?

If not, @lime124 can we remove this from the board, since it looks like this isn't particularly actionable right now, and should be represented by other bugs? (or even close it)

**Comment by bifleming** [comment]
Per planning meeting, will close this item. Will file another ticket under the 're-design meta' to address animation issues.
